### PR TITLE
Feature/fixing languages

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SettingsMenuFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5243367&bid=17186254)

**Review by enrichp — APPROVED** [review]

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java [hunk 1]
```diff
@@ -81,7 +81,7 @@
         SettingsMenuFragment.class
     };
 
-    private static boolean enableLoginActivity = true;
+    private static boolean enableLoginActivity = false;
```
Comment (xavier-campos) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java]: Check this out, enableLoginActivity should be true

Comment (xavier-campos-diaz) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java]: Check this out!!! 
enableLoginActivity should be true

**Review by xavier-campos-diaz — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SettingsMenuFragment.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5243367&bid=17186254)

**Pull request merged**